### ISSUE TITLE
All calc and some data loader tests failing on v0.1 installed via conda-forge

### ISSUE THREAD

**Issue by spencerahill** [spencerahill/aospy#131]
Tried for both python 2.7 and python 3.6.  Have confirmed that I'm in a clean environment, i.e. this isn't stemming from accidentally using pytest or aospy from wrong location.

On 2.7, there's additionally a name error from `FileNotFoundError`.  Other than this, the failures are all the same, occurring during the variable renaming.

```
(aospy27) shill@shillbook:~$ py.test /Users/shill/Dropbox/miniconda3/envs/aospy27/lib/python2.7/site-packages/aospy
================================================================= test session starts ==================================================================
platform darwin -- Python 2.7.12, pytest-3.0.6, py-1.4.31, pluggy-0.4.0
rootdir: /Users/shill, inifile:
collected 86 items

Dropbox/miniconda3/envs/aospy27/lib/python2.7/site-packages/aospy/test/test_calc_basic.py FEFEFEFEFEFEFEFEFEFEFEFEFEFEFEFEFEFE
Dropbox/miniconda3/envs/aospy27/lib/python2.7/site-packages/aospy/test/test_constant.py ....ss.......x
Dropbox/miniconda3/envs/aospy27/lib/python2.7/site-packages/aospy/test/test_data_loader.py ...F....F....F.............F.
Dropbox/miniconda3/envs/aospy27/lib/python2.7/site-packages/aospy/test/test_run.py ...
Dropbox/miniconda3/envs/aospy27/lib/python2.7/site-packages/aospy/test/test_utils_io.py .
Dropbox/miniconda3/envs/aospy27/lib/python2.7/site-packages/aospy/test/test_utils_times.py ..................
Dropbox/miniconda3/envs/aospy27/lib/python2.7/site-packages/aospy/test/test_utils_vertcoord.py ...

======================================================================== ERRORS ========================================================================
_________________________________________________ ERROR at teardown of TestCalcBasic.test_annual_mean __________________________________________________

self = <aospy.test.test_calc_basic.TestCalcBasic testMethod=test_annual_mean>

    def tearDown(self):
        for direc in [example_proj.direc_out, example_proj.tar_direc_out]:
            try:
                shutil.rmtree(direc)
>           except FileNotFoundError:
E           NameError: global name 'FileNotFoundError' is not defined

Dropbox/miniconda3/envs/aospy27/lib/python2.7/site-packages/aospy/test/test_calc_basic.py:32: NameError
----------------------------------------------------------------- Captured stderr call -----------------------------------------------------------------
INFO:root:Initializing Calc instance: Calc object: condensation_rain, example_proj, example_model, example_run (Mon Jan 30 14:04:50 2017)
Exception AttributeError: "'netcdf_file' object has no attribute 'fp'" in <bound method netcdf_file.close of <scipy.io.netcdf.netcdf_file object at 0x10a433710>> ignored
INFO:root:Getting input data: Var instance "condensation_rain" (Mon Jan 30 14:04:50 2017)
WARNING:root:dmget command not found in this machine
/Users/shill/Dropbox/miniconda3/envs/aospy27/lib/python2.7/site-packages/aospy/utils/times.py:198: FutureWarning: the order of the arguments on DataArray.to_dataset has changed; you now need to supply ``name`` as a keyword argument
  min_yr_decoded = xr.decode_cf(time.to_dataset('dummy'))
/Users/shill/Dropbox/miniconda3/envs/aospy27/lib/python2.7/site-packages/xarray/conventions.py:389: RuntimeWarning: Unable to decode time axis into full numpy.datetime64 objects, continuing using dummy netCDF4.datetime objects instead, reason: dates out of range
  result = decode_cf_datetime(example_value, units, calendar)
/Users/shill/Dropbox/miniconda3/envs/aospy27/lib/python2.7/site-packages/xarray/conventions.py:408: RuntimeWarning: Unable to decode time axis into full numpy.datetime64 objects, continuing using dummy netCDF4.datetime objects instead, reason: dates out of range
  calendar=self.calendar)
```

Compare to 3.6:

```
(aospy) shill@shillbook:~/Dropbox/miniconda3/envs/aospy/lib/python3.6/site-packages/aospy$ py.test test/
================================================================= test session starts ==================================================================
platform darwin -- Python 3.6.0, pytest-3.0.6, py-1.4.32, pluggy-0.4.0
rootdir: /Users/shill/Dropbox/miniconda3/envs/aospy/lib/python3.6/site-packages/aospy, inifile:
collected 86 items

test/test_calc_basic.py FFFFFFFFFFFFFFFFFF
test/test_constant.py ....ss.......x
test/test_data_loader.py ...F....F....F.............F.
test/test_run.py ...
test/test_utils_io.py .
test/test_utils_times.py ..................
test/test_utils_vertcoord.py ...

======================================================================= FAILURES =======================================================================
____________________________________________________________ TestCalcBasic.test_annual_mean ____________________________________________________________

self = <aospy.test.test_calc_basic.TestCalcBasic testMethod=test_annual_mean>

    def test_annual_mean(self):
        calc_int = CalcInterface(intvl_out='ann',
                                 dtype_out_time='av',
                                 **self.two_d_test_params)
        calc = Calc(calc_int)
>       calc.compute()

test/test_calc_basic.py:40:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
calc.py:660: in compute
    self.end_date),
calc.py:478: in _get_all_data
    for n, var in enumerate(self.variables)]
calc.py:478: in <listcomp>
    for n, var in enumerate(self.variables)]
calc.py:425: in _get_input_data
    **self.data_loader_attrs)
data_loader.py:186: in load_variable
    da = _sel_var(ds, var)
data_loader.py:85: in _sel_var
    return da.rename({name: var.name})
../xarray/core/dataarray.py:814: in rename
    dataset = self._to_temp_dataset().rename(new_name_or_name_dict)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <xarray.Dataset>
Dimensions:         (lat: 64, lon: 128, time: 36)
Coordinates:
  * lon             (lon) float64 0.0 ...te    datetime64[ns] 1681-01-01
Data variables:
    <this-array>    (time, lat, lon) float32 5.7683e-06 5.78355e-06 ...
name_dict = {'condensation_rain': 'condensation_rain'}, inplace = False

    def rename(self, name_dict, inplace=False):
        """Returns a new object with renamed variables and dimensions.

            Parameters
            ----------
            name_dict : dict-like
                Dictionary whose keys are current variable or dimension names and
                whose values are the desired names.
            inplace : bool, optional
                If True, rename variables and dimensions in-place. Otherwise,
                return a new dataset object.

            Returns
            -------
            renamed : Dataset
                Dataset with renamed variables and dimensions.

            See Also
            --------
            Dataset.swap_dims
            DataArray.rename
            """
        for k, v in name_dict.items():
            if k not in self and k not in self.dims:
                raise ValueError("cannot rename %r because it is not a "
>                                "variable or dimension in this dataset" % k)
E               ValueError: cannot rename 'condensation_rain' because it is not a variable or dimension in this dataset

../xarray/core/dataset.py:1506: ValueError
----------------------------------------------------------------- Captured stderr call -----------------------------------------------------------------
INFO:root:Initializing Calc instance: Calc object: condensation_rain, example_proj, example_model, example_run (Mon Jan 30 13:59:59 2017)
Exception ignored in: <bound method netcdf_file.close of <scipy.io.netcdf.netcdf_file object at 0x108a9eeb8>>
Traceback (most recent call last):
  File "/Users/shill/Dropbox/miniconda3/envs/aospy/lib/python3.6/site-packages/scipy/io/netcdf.py", line 277, in close
    if not self.fp.closed:
AttributeError: 'netcdf_file' object has no attribute 'fp'
INFO:root:Getting input data: Var instance "condensation_rain" (Mon Jan 30 14:00:00 2017)
WARNING:root:dmget command not found in this machine
/Users/shill/Dropbox/miniconda3/envs/aospy/lib/python3.6/site-packages/aospy/utils/times.py:198: FutureWarning: the order of the arguments on DataArray.to_dataset has changed; you now need to supply ``name`` as a keyword argument
  min_yr_decoded = xr.decode_cf(time.to_dataset('dummy'))
/Users/shill/Dropbox/miniconda3/envs/aospy/lib/python3.6/site-packages/xarray/conventions.py:389: RuntimeWarning: Unable to decode time axis into full numpy.datetime64 objects, continuing using dummy netCDF4.datetime objects instead, reason: dates out of range
  result = decode_cf_datetime(example_value, units, calendar)
/Users/shill/Dropbox/miniconda3/envs/aospy/lib/python3.6/site-packages/xarray/conventions.py:408: RuntimeWarning: Unable to decode time axis into full numpy.datetime64 objects, continuing using dummy netCDF4.datetime objects instead, reason: dates out of range
  calendar=self.calendar)
```

**Comment by spencerkclark** [comment]
@spencerahill this is just compatibility with xarray 0.9.0, and fixed in #128.  We should delete that exception block entirely; we always want the tests to fail if the files don't exist anyway.  I'll take care of that soon.

**Comment by spencerahill** [comment]
Cool, thanks!

**Comment by spencerahill** [comment]
Closed by #128
